src/warnet/warnet.py

```diff
@@ -122,6 +127,43 @@ def tanks_from_graph(self):
                 )
             self.tanks.append(Tank.from_graph_node(node_id, self))
         logger.info(f"Imported {len(self.tanks)} tanks from graph")
+        self.generate_as_map()
+
+    def generate_as_map(self):
+        # Write AS mappings to file
+        with open(self.as_map_txt_path, "w") as f:
+            for tank in self.tanks:
+                f.write(f"{tank.ipv4}/32 AS{tank.autonomous_system}\n")
```

Comment:
why "/32" here? will this /32 work regardless of where warnet is running?

Reply:
That's means "a single ip address"